docs/datagrid/columns.md

```diff
@@ -65,6 +25,57 @@ columns: {
 }
 ```
 
+### Customize headers
+By default, all columns from the DataTable are imported into the DataGrid in the order they are declared.
+This can be modified using the [settings.header](https://api.highcharts.com/dashboards/#interfaces/DataGrid_DataGridOptions.ColumnsSetting#header) option to define columns (or order) that you would like ot display.
+
+```js
+settings: {
+  header: ['a', 'price', 'weight']
```

Comment:
IMO we should link to a simple demo here- it could be also used in API.

Reply:
I think that the grouped headers is enough. In the demo we include string using as well.